aws/gemini-3g/main.tf

```diff
@@ -0,0 +1,114 @@
+    genesis-hash       = "92e91e657747c41eeabed5129ff51689d2e935b9f6abfbd5dfcb2e1d0d035095"
+    dsn-listen-port    = 30533
+    node-dsn-port      = 30433
+    disk-volume-size   = var.disk_volume_size
+    disk-volume-type   = var.disk_volume_type
+  }
+
+  bootstrap-node-evm-config = {
+    instance-type      = var.instance_type["evm_bootstrap"]
+    deployment-version = 0
+    regions            = var.aws_region
+    instance-count     = var.instance_count["evm_bootstrap"]
+    docker-org         = "subspace"
+    docker-tag         = "gemini-3g-2023-oct-31"
+    reserved-only      = false
+    prune              = false
+    genesis-hash       = "92e91e657747c41eeabed5129ff51689d2e935b9f6abfbd5dfcb2e1d0d035095"
+    dsn-listen-port    = 30533
+    node-dsn-port      = 30433
+    operator-port      = 40333
+    disk-volume-size   = var.disk_volume_size
+    disk-volume-type   = var.disk_volume_type
+  }
+
+  full-node-config = {
+    instance-type      = var.instance_type["full"]
+    deployment-version = 0
+    regions            = var.aws_region
+    instance-count     = var.instance_count["full"]
+    docker-org         = "subspace"
+    docker-tag         = "gemini-3g-2023-oct-31"
+    reserved-only      = true
+    prune              = false
+    node-dsn-port      = 30433
+    disk-volume-size   = var.disk_volume_size
+    disk-volume-type   = var.disk_volume_type
+  }
+
+  rpc-node-config = {
+    instance-type      = var.instance_type["rpc"]
+    deployment-version = 0
+    regions            = var.aws_region
+    instance-count     = var.instance_count["rpc"]
+    docker-org         = "subspace"
+    docker-tag         = "gemini-3g-2023-oct-31"
+    domain-prefix      = "rpc"
+    reserved-only      = true
+    prune              = false
+    node-dsn-port      = 30433
+    disk-volume-size   = var.disk_volume_size
+    disk-volume-type   = var.disk_volume_type
+  }
+
+  domain-node-config = {
+    instance-type      = var.instance_type["domain"]
+    deployment-version = 0
+    regions            = var.aws_region
+    instance-count     = var.instance_count["domain"]
+    docker-org         = "subspace"
+    docker-tag         = "gemini-3g-2023-oct-31"
+    domain-prefix      = "nova"
+    reserved-only      = true
+    prune              = false
+    node-dsn-port      = 30434
+    enable-domains     = true
+    domain-id          = var.domain_id
+    domain-labels      = var.domain_labels
+    disk-volume-size   = var.disk_volume_size
+    disk-volume-type   = var.disk_volume_type
+  }
+
+  farmer-node-config = {
+    instance-type          = var.instance_type["farmer"]
+    deployment-version     = 0
+    regions                = var.aws_region
+    instance-count         = var.instance_count["farmer"]
+    docker-org             = "subspace"
+    docker-tag             = "gemini-3g-2023-oct-31"
+    reserved-only          = true
+    prune                  = false
+    plot-size              = "10G"
```

Comment:
Didn't we deply with 2G instead?

Reply:
Actually yes, I think it should be sufficient to bootstrap the network. I just don't like when things deployed to servers and in `main` diverging.